ccube.nim

```diff
@@ -14,23 +14,23 @@ Distributed under GPLV3
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ]#
-import std/[os,strutils,strformat,random,json]
+import std/[os, strutils, strformat, random, json, parseopt]
 import bigints
 
 randomize()
 
-let VER = "1.0.0"
-let banner = fmt"""
+const VER = "1.0.0"
+const banner = fmt"""
 ░█████╗░░█████╗░██╗░░░██╗██████╗░███████╗
 ██╔══██╗██╔══██╗██║░░░██║██╔══██╗██╔════╝
 ██║░░╚═╝██║░░╚═╝██║░░░██║██████╦╝█████╗░░
 ██║░░██╗██║░░██╗██║░░░██║██╔══██╗██╔══╝░░
 ╚█████╔╝╚█████╔╝╚██████╔╝██████╦╝███████╗
 ░╚════╝░░╚════╝░░╚═════╝░╚═════╝░╚══════╝Version:{VER}"""
 
-proc showUsage():void =  
```

Comment:
`void` return type is not needed in Nim, same for the parenthesis if there are no arguments